joomla/joomla-cms
# [4.2] Replace Factory::getUser() with $this->getCurrentUser() in components view class

Pull Request for Issue # .

### Summary of Changes
With https://github.com/joomla/joomla-cms/pull/37498/ being merged into 4.2, we can now replace deprecated method `Factory::getUser()` in components view classes with new method `$this->getCurrentUser()`

### Testing Instructions
1. Use Joomla 4.2 nightly build
2. Apply patch
3. Access to random pages (frontend, backend), make sure nothing is broken.
